ui/app/helpers/utils/util.js

```diff
@@ -351,3 +352,8 @@ function isValidAddressHead (address) {
 
   return addressLengthIsLessThanFull && addressIsHex
 }
+
+function getOriginFromUrl (url) {
+  const match = url.match(/\/\/([^/:]+)\/?/)
```

Comment:
Could [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) be used here instead?

Reply:
Change made in 808813653